gdnative-derive/src/methods.rs

```diff
@@ -82,7 +83,10 @@ pub(crate) fn derive_methods(item_impl: ItemImpl) -> TokenStream2 {
             let sig_span = sig.ident.span();
 
             let name = sig.ident;
-            let name_string = name.to_string();
+            let name_string = match args.name_override {
+                Some(name_override) => name_override,
+                None => name.to_string(),
+            };
```

Comment:
Could be written more concisely as `args.name_override.unwrap_or_else(|| name.to_string())`, I believe?

Reply:
Much better, thanks!